tools/edge_cut/src/main.cpp

```diff
@@ -50,55 +70,121 @@ int main( int i_argc, char *i_argv[] ) {
+  // NOTE A "Surface Delaunay Ball" is the 3D ball circumscribing a triangle surface facet which also
+  //      has its center on the theoretical surface to be meshed. The center of the Surface
+  //      Delaunay Ball will not coincide with the triangle circumcenter when the surface mesh
+  //      is a poor approximation to the theoretical surface
+  K::FT l_scale = std::stod( doc.child("region").child_value("scale") );
+  K::FT l_innerRefineRad = std::stod( doc.child("region").child_value("inner_rad") );
+  K::FT l_outerRefineRad = std::stod( doc.child("region").child_value("outer_rad") );
+  K::Point_3 l_center = K::Point_3( std::stod( doc.child("region").child("center").child_value("x") ),
+                                    std::stod( doc.child("region").child("center").child_value("x") ),
+                                    std::stod( doc.child("region").child("center").child_value("x") ) );
+
+  K::FT l_edgeLengthBase  = std::stod( doc.child("refine").child_value("edge") );
+  K::FT l_facetSizeBase   = std::stod( doc.child("refine").child_value("facet") );
+  K::FT l_facetApproxBase = std::stod( doc.child("refine").child_value("approx") );
+  K::FT l_angleBound      = std::stod( doc.child("refine").child_value("angle") );
+
+  // NOTE meshes must share common edge refinement criteria in order for borders
+  //      to coincide. (recall edge criteria only affects specified 1D features)
+  SizingField l_edgeCrit( l_edgeLengthBase, l_scale, l_center, l_innerRefineRad, l_outerRefineRad );
+
+  SizingField l_topoFacetCrit( l_facetSizeBase, l_scale, l_center, l_innerRefineRad, l_outerRefineRad );
+  SizingField l_bdryFacetCrit( l_facetSizeBase, l_scale, l_center, 0, 0 );
+  SizingField l_topoApproxCrit( l_facetApproxBase, l_scale, l_center, l_innerRefineRad, l_outerRefineRad );
+  SizingField l_bdryApproxCrit( l_facetApproxBase, l_scale, l_center, 0, 0 );
+
+  Mesh_criteria   l_topoCriteria( CGAL::parameters::edge_size = l_edgeCrit,
+                                  CGAL::parameters::facet_size = l_topoFacetCrit,
+                                  CGAL::parameters::facet_distance = l_topoApproxCrit,
+                                  CGAL::parameters::facet_angle = l_angleBound         );
+  Mesh_criteria   l_bdryCriteria( CGAL::parameters::edge_size = l_edgeCrit,
+                                  CGAL::parameters::facet_size = l_bdryFacetCrit,
+                                  CGAL::parameters::facet_distance = l_bdryApproxCrit,
+                                  CGAL::parameters::facet_angle = l_angleBound         );
+
+  // Mesh generation
+  // NOTE the optimizers have more options than specified here
+  EDGE_LOG_INFO << "Re-meshing polyhedral surfaces according to provided criteria";
+  C3t3 topoComplex = CGAL::make_mesh_3<C3t3>( l_topoDomain,
+                                              l_topoCriteria,
+                                              CGAL::parameters::lloyd(    CGAL::parameters::time_limit = 60 ),
```

Comment:
What is ``CGAL::parameters::time_limit`` doing? Should be a runtime-parameters, for example, be part of a <cgal/> section in the config.